genie-common-internal/src/main/java/com/netflix/genie/common/internal/services/impl/JobArchiveServiceImpl.java

```diff
@@ -52,8 +56,22 @@ public JobArchiveServiceImpl(final List<JobArchiver> jobArchivers) {
      */
     @Override
     public void archiveDirectory(final Path directory, final URI target) throws JobArchiveException {
+        // Create a manifest and write it to the root of the job directory
```

Comment:
I would not put it in root, since it's an "internal" file, i'd put it in `genie/`.

Reply:
A fair point. I thought about this too. I figured since it would only live in the archive location and never be seen by the user it didn't really matter and I wasn't sure we were keeping the `genie` directory around in 4. I can put it there for now. We can move it later if need be.

Reply:
I've put this in `genie/`. I don't really love the implementation but I want to get this merged. Will likely want to revisit at some point in the near future.